code/modules/antagonists/heretic/magic/nightwatcher_rebirth.dm

```diff
@@ -44,17 +54,26 @@
 		victim.investigate_log("has been executed by fiery rebirth.", INVESTIGATE_DEATHS)
 		victim.death()
 	victim.apply_damage(20, BURN)
+	victim.extinguish_mob()
 
 	// Heal the caster for every victim damaged
 	var/need_mob_update = FALSE
-	need_mob_update += caster.adjustBruteLoss(-10, updating_health = FALSE)
-	need_mob_update += caster.adjustFireLoss(-10, updating_health = FALSE)
-	need_mob_update += caster.adjustToxLoss(-10, updating_health = FALSE)
-	need_mob_update += caster.adjustOxyLoss(-10, updating_health = FALSE)
-	need_mob_update += caster.adjustStaminaLoss(-10, updating_stamina = FALSE)
+	need_mob_update += caster.adjustBruteLoss(-20, updating_health = FALSE)
+	need_mob_update += caster.adjustFireLoss(-20, updating_health = FALSE)
+	need_mob_update += caster.adjustToxLoss(-20, updating_health = FALSE, forced = TRUE)
+	need_mob_update += caster.adjustOxyLoss(-20, updating_health = FALSE)
+	need_mob_update += caster.adjustStaminaLoss(-20, updating_stamina = FALSE)
 	if(need_mob_update)
 		caster.updatehealth()
 
+/datum/action/cooldown/spell/aoe/fiery_rebirth/after_cast(atom/cast_on)
+	. = ..()
+	if(!victims_counter)
+		StartCooldown(cooldown_time)
+		return
+	var/new_cooldown = cooldown_time - victims_counter * 10 SECONDS
+	StartCooldown(max(9 SECONDS, new_cooldown)) // Hard capped so an ascended heretic doesn't get to freely spam
```

Comment:
I think either the healing, radius, or the cooldown be buffed, but all three at once. Pretty unfun

Reply:
holy fuck 20 healing per target, that's basically a full heal

Reply:
undid the buffs besides the CD